classes/firmware-qcom-sign.bbclass

```diff
@@ -0,0 +1,133 @@
+sign_verify_file () {
+	file="$(basename $1)"
+	filedir="$(dirname "$1")"
+	sectools="${QCOM_FIRMWARE_SIGN_SECTOOLS_DIR}/sectools"
```

Comment:
We should probably verify and abort in case it can't be found.

Reply:
It was already verified on the function do_firmware_sign.

Reply:
True, it is checking everything from the folder.